### ISSUE TITLE
Mod Profiles

### ISSUE THREAD

**Issue by bgirton29** [bluemagic123/tModLoader#2029]
### Do you intend to personally contribute/program this feature?

Yes

### Description

Make it so that way you can have specific mod profiles that are isolated from each other, similar to like mc modpacks in a way.

### What does this proposal attempt to solve or improve?

I feel like installing and uninstalling mods with enabling and disabling modpacks is too much of a hassle. So you can have the mods you want/need in different isolated profile like mc modpacks.

**Comment by JavidPack** [comment]
How would this function differently from the current modpack feature/menu? I'm a little lost

**Comment by bgirton29** [comment]
> How would this function differently from the current modpack feature/menu? I'm a little lost

Think of it like minecraft modpacks. Where you can create new modpacks with no mods in them, and you can run it without other modpacks interfering with your current one.

**Comment by JavidPack** [comment]
Yeah, that's what the modpacks menu is for. You can make 1 modpack for a personal playthrough, and another modpack for a set of mods you are playing with your friends, for example. Use the menu to toggle between them.

If you don't know where that menu is, it's a button in the Mods menu

**Comment by bgirton29** [comment]
> 

I see your point. This is definitely a mod more for me. Is there a way where I can change the modpack ui manually? If not that would be something that I would like to see implemented in 1.4

**Comment by JavidPack** [comment]
I imagine someone determined could figure out a mod for manipulating the UI in some way. Its not something that will be answerable here, but some mod authors have done small adjustments in their mods and might be willing to give you some pointers on the tmodloader discord if they can.
